### ISSUE TITLE
Unable to scroll menu's on laptops that don't have a dedicated pgup/down key

### ISSUE THREAD

**Issue by Givemeurcookies** [CleverRaven/Cataclysm-DDA#28012]
**Describe the bug**  
I've gone through a lot of playtime on CDDA now but there's on particual issue that's causing great annoyance, being unable to scroll in menu's when inspecting items (i.e weapons has important info that you have to scroll to get at). I'm playing on a laptop which doesn't use the typical full keyboard layout that comes with desktops. It does sport a keypad though but the keypad PG_UP and PG_DOWN does not work. Mousepad also sends a SCROLL_UP and SCROLL_DOWN key signal but this also doesn't work.

**How to reproduce**
1. Inspect item with long description, i.e weapon.
2. Try to scroll without using dedicated PG_UP and PG_DOWN keys. 

**Expected behavior**  
I'd expect to have a way to scroll menu's. Not being able to remap or add to the mapping of scroll this just adds to the issue.

**Versions and configuration(please complete the following information):**
OS: Windows 10
Game Version: 0.C-8487 (0.C-37192-gf53995c). Experimental build, pre-complied.
Graphics version: Tiles
Mods loaded: DDA, modular turrets, salvaged robots, disable npc needs, filthy clothing, simplified nutrition, alternative map key, boats, mutant npcs.

**Additional context**  
There's already a few "duplicates" of this issue (i.e #14647), however they've been closed without being fixed several years and just been referenced to a bigger UI consistency document (#12112). I believe there shall be some way to scroll on a laptop keyboard that does not sport dedicated keys.

**Comment by gcaliber** [comment]
I believe scrolling the item info on these screens doesn't work at all. There are bindable keys to scroll item descriptions from the pickup menu, but no option for any such keybinds when examining an item.

**Comment by Asmageddon** [comment]
Having just tested this:
 * In examine menu, it's possible to rebind these keys, and they work
 * In the crafting menu, it's possible to rebind these keys and they work
 * In inventory, examining an item, only hardcoded Page Down and Page Up work, changing the 4 scrolling keybindings is actually related to moving the action list's selection.
 * In advanced inventory, only hardcoded Page Down and Page Up work, and there are no related keybindings.

Furthermore, as #12112 points out, none of those bindings are even the same as in other menus.

**Comment by Elvecent** [comment]
Is there any workaround other than using an on-screen keyboard?

**Comment by Night-Pryanik** [comment]
Examining items through `i` -> `e`: one can rebind `Page up` and `Page down`.
Examining items through `i` -> `enter` and through AIM: one can rebind `Fast scroll up` and `Fast scroll down`.

Furthermore, after #60006 will be merged, it will be possible to use mouse to drag scrollbars in item descriptions.

I think this issue could be closed.

**Comment by ZeroInternalReflection** [comment]
> Furthermore, after #60006 will be merged, it will be possible to use mouse to drag scrollbars in item descriptions.

Please note that #60006 is essentially a proof of concept.  It only allows click-and-drag of the scrollbar on things using the "scrolling_text_view" helper class.  I believe item descriptions typically use draw_item_info, which would be high on my to-do list for implementing click-and-drag, but would be a separate PR.
